CHANGES.rst

```diff
@@ -20,6 +20,9 @@ Release history
 
 3.4.5 (unreleased)
 ------------------
+**Added**
+
+- Added ``groups`` parameter to ConvertConv
```

Comment:
Should have a period at the end, and link to the pull request. See the other entries in this file for examples.